flow360/component/simulation/unit_system.py

```diff
@@ -552,6 +562,14 @@ def Vector(self):
         """
         return self._VectorType.get_class_object(self)
 
+    # pylint: disable=invalid-name
+    @classproperty
+    def PositiveVector(self):
```

Comment:
Can we have a better name for it? The length is restricted to 3 right?

Reply:
Keeping it as is for now.